globals.go

```diff
@@ -902,6 +902,22 @@ var (
 		HostStopped,
 	}
 
+	// StoppableHostStatuses represent all host statuses when it is possible to
```

Comment:
I think the PR explains well why these might get in wonky statuses, and I can think of reasons why it'd happen, like if the user decided to start the instance early we wouldn't want to "fail" to start it. Could we add some detail here to explain? Just looking at these lists, it seems like an accident that "stopped" is in stoppable and "running" is in startable.

Reply:
Done, added some comments for each specific state.